client/src/docs-ui/menu/menu.style.ts

```diff
@@ -3,8 +3,9 @@ import {css} from "emotion";
 export const menuStyle = css`
   display: block;
   padding: 0 2.5rem;
-  overflow-y: overlay;
-  padding-bottom: 6rem;
+  overflow-y: scroll; /* for Firefox */
+  overflow-y: overlay; /* for Webkit browsers */
```

Comment:
Have you verified that line 6 doesn't get overwritten by line 7 on firefox?

Reply:
yeah, it just gets totally removed, which is admittedly pretty weird...